### PR TITLE
Release 1.51.5

### PR DESCRIPTION
**Pull request by scrosby**
Please rebase but do not squash.

## Changes proposed in this PR

- Release Cook

### CODE REVIEW

File: scheduler/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.51.5] - 2021-03-04
```
Comment (dposada) [inline scheduler/CHANGELOG.md:7]: should be 2021-03-08

File: scheduler/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file
...
+### Changed
+- A knob letting Cook clobber syhthetic pods with real jobs for k8s, @scrosby
```
Comment (dposada) [inline scheduler/CHANGELOG.md:9]: should be "from @scrosby"

File: scheduler/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file
...
+
```
Comment (dposada) [inline scheduler/CHANGELOG.md:10]: This is missing PR #1805

File: scheduler/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.51.5] - 2021-03-08
+### Changed
+- A knob letting Cook clobber syhthetic pods with real jobs for k8s, from @scrosby
+### Fixed
+- Look for Cook memory labels on job labels, not pod labels, from nsinkov
```
Comment (dposada) [inline scheduler/CHANGELOG.md:11]: should be "@nsinkov"

Comment (scrosby) [reply scheduler/CHANGELOG.md:11]: Thanks.

**Pull request merged**